core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -229,7 +229,8 @@ object Schema extends SchemaExtensions with LowPrioritySchema with SchemaCompani
   implicit val schemaForDouble: Schema[Double] = Schema(SNumber[Double]()).format("double")
   implicit val schemaForBoolean: Schema[Boolean] = Schema(SBoolean())
   implicit val schemaForUnit: Schema[Unit] = Schema(SProduct.empty)
-  implicit val schemaForFile: Schema[TapirFile] = Schema(SBinary())
+  implicit val schemaForNormalFile: Schema[File] = Schema(SBinary())
```

Comment:
we can't use java.io.File in jvm/js shared sources - this needs to go to `SchemaExtensions`